orchagent/bfdorch.cpp

```diff
@@ -62,16 +64,27 @@ BfdOrch::BfdOrch(DBConnector *db, string tableName, TableConnector stateDbBfdSes
     m_bfdStateNotificationConsumer = new swss::NotificationConsumer(notificationsDb, "NOTIFICATIONS");
     auto bfdStateNotificatier = new Notifier(m_bfdStateNotificationConsumer, this, "BFD_STATE_NOTIFICATIONS");
 
-    // Clean up state database BFD entries
+    m_stateDbConnector = std::make_unique<swss::DBConnector>("STATE_DB", 0);
+    m_stateSoftBfdSessionTable = std::make_unique<swss::Table>(m_stateDbConnector.get(), STATE_BFD_SOFTWARE_SESSION_TABLE_NAME);
+
+    SWSS_LOG_NOTICE("Switch type is: %s", gMySwitchType.c_str());
+
     vector<string> keys;
 
+    // Clean up state database BFD entries
     m_stateBfdSessionTable.getKeys(keys);
-
     for (auto alias : keys)
     {
         m_stateBfdSessionTable.del(alias);
     }
-
+    // Clean up state database software BFD entries
+    if (gMySwitchType == "dpu") {
+        m_stateSoftBfdSessionTable->getKeys(keys);
```

Comment:
We don't have to check for 'dpu' here. Clean it anyways based on `getKeys `function